src/Javis.jl

```diff
@@ -763,6 +817,26 @@ get_position(s::Symbol) = get_position(val(s))
 """
 pos(x) = get_position(x)
 
+# As it is just a number we can return it
+get_scale(x::Number) = x
+
+"""
+    get_scale(s::Symbol)
+
+Get access to the sclaing that got saved in `s` by a previous action.
+
+# Returns
+- `Scaling`: the scale stored by a previous action.
+"""
+get_scale(s::Symbol) = get_scale(val(s))
+
+"""
+    scl(x)
+
+`scl` is just a short-hand for [`get_scale`](@ref)
+"""
+scl(x) = get_scale(x)
```

Comment:
Why do we want this short hand? I feel like the `scl` is not as explicit as the standard `get_scale`.

Reply:
I like short hands :smile: I kind of enjoy that we have the 3 letter forms `pos`, `ang` and `scl`. Actually I'm not sure whether we need the `scl` and `get_scale` functionality as it basically saves just the val so we can access it with `val` / `get_value` and I don't have a good use case for it any way.

Reply:
Haha, ok, ok, we can keep it. :wink: however, if you don't think it has any useful functionality, we can just remove it.

Reply:
I think we do need them as scale supports a tuple and a single `Real` and needs to convert a single `Real` to a tuple.

Reply:
Sounds good. Resolved